src/App.js

```diff
@@ -1,24 +1,14 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import './styles/App.css';
+import ItemsList from './Components/ItemsList';
```

Comment:
I think this should be './components/ItemsList' to match the case of the file path, but I've mentioned in another comment below that in our PR we have used a capital C so I think we just need to agree on the casing 😊

Reply:
I think this is why the Netlify checks are failing at the moment also, hopefully that should fix it! 🎉

Reply:
Fixed in https://github.com/the-collab-lab/tcl-19-smart-shopping-list/pull/18/commits/b7817e3af92341534a25090c577ad6c7e639b9a6